docs/source/conf.py

```diff
@@ -26,7 +26,7 @@
 author = authors
 copyright = f"2023, {authors}"
 # The full version, including alpha/beta/rc tags
-release = "0.1.0"
+release = "1.0.0"
```

Comment:
Not a problem for this PR, but I'll write it here so I don't forget as I review. Having to change the version string in two places feels like a recipe to forget one of them. Is it worth working out how to pull the version string out of the pyproject.toml file to use in the documentation configuration?

Reply:
Fixed in subsequent commits